src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/SqlConnectionString.cs

```diff
@@ -1117,8 +1112,7 @@ internal bool ConvertValueToEncrypt()
         {
             // If the Authentication keyword is provided, default to Encrypt=true;
             // otherwise keep old default for backwards compatibility
-            object authValue = base.Parsetable[KEY.Authentication];
-            bool defaultEncryptValue = (authValue == null) ? DEFAULT.Encrypt : true;
+            bool defaultEncryptValue = !base.Parsetable.ContainsKey(KEY.Authentication) ? DEFAULT.Encrypt : true;
```

Comment:
The comment here is not right now(after #1210, 4.0.0-preview1), now `defaultEncryptValue` is always true

Reply:
done